packages/paste-codemods/tools/mappings.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
Is there a requirement for this file to be checked in? Can it just behave like a build asset, similar to a dist js file? That way we don't need to keep updating it in PRs that introduce new packages?